docs/content/policy-language.md

```diff
@@ -2890,7 +2894,8 @@ allow if {
 ### Entrypoint
 
 The `entrypoint` annotation is a boolean used to mark rules and packages that should be used as entrypoints for a policy.
-This value is false by default, and can only be used at `rule` or `package` scope.
+This value is false by default, and can only be used at `document` or `package` scope. When used on a rule with no
+explicit `scope` set, the presence of an `entrypoint` annotation will automatically set the scope to `document`.
```

Comment:
I just noticed this section doesn't have an example. Should we add one since we're here already? Would be an opportunity to make the `scope` implications even more obvious.

```
# METADATA
# entrypoint: true
allow if {
  ...
}

# NOTE: The METADATA block on the above rule has an implicit scope of 'document' and also applies to this rule
allow if {
  ...
}
```

Reply:
Sure! I'll add an example